contracts/sources/lending_market.move

```diff
@@ -379,6 +379,30 @@ module suilend::lending_market {
         obligation::zero_out_rewards_if_looped(obligation, &mut lending_market.reserves, clock);
         coin::from_balance(receive_balance, ctx)
     }
+    
+    /// Set emode for obligation - T is the deposit coin type
+    public fun set_emode<P>(
```

Comment:
i don't think we need to know the reserve array indicies to set emode?

Reply:
an obligation is a valid emode obligation if it has 1 or less deposits and 1 or less borrows

Reply:
When we call deposit for an emode obligation we need to be able to know what the corresponding borrow reserve is in order to get the respective emode ltv